docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,133 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
+ * configure networking in finish template ([#9400](http://projects.theforeman.org/issues/9400), [2df454b](http://github.com/katello/katello/commit/2df454b))
+
+### Capsule
+ * fixing capsule sync timeout (cherry picked from commit 3a7238da41c603581f59e3cd3aad620f68225e99) ([#7162](http://projects.theforeman.org/issues/7162), [85be9e3](http://github.com/katello/katello/commit/85be9e3))
+
+### API
+ * Adding docker_images method to cvv (cherry picked from commit 5aae5f8ded418d29bbfc994d89723e19561d325a) ([#9755](http://projects.theforeman.org/issues/9755), [11d2894](http://github.com/katello/katello/commit/11d2894))
+ * verify errata exist before application ([#9502](http://projects.theforeman.org/issues/9502), [#9503](http://projects.theforeman.org/issues/9503), [6a3a85a](http://github.com/katello/katello/commit/6a3a85a))
+
+### Errata Management
+ * fixing race condition on errata index ([#8586](http://projects.theforeman.org/issues/8586), [d379aca](http://github.com/katello/katello/commit/d379aca))
+
+### Subscriptions
+ * handle case where content host no longer exists ([#10514](http://projects.theforeman.org/issues/10514), [1874ba1](http://github.com/katello/katello/commit/1874ba1))
+ * Backend cleanup script handles uebercert and hypervisors. ([#9915](http://projects.theforeman.org/issues/9915), [3edb3a8](http://github.com/katello/katello/commit/3edb3a8))
+
+### Database
+ * Ensure content host exists before updating in migration. (cherry picked from commit 1136484750f1d8b882406cabaf0043fd6692ad39) ([#10426](http://projects.theforeman.org/issues/10426), [3137ab0](http://github.com/katello/katello/commit/3137ab0))
+
+### Upgrades
+ * fixing error with import_errata rake task (cherry picked from commit bf6ec73c81ec9329bd0b7f2ca26edc2dc8d8e0ce) ([#9354](http://projects.theforeman.org/issues/9354), [1ef1883](http://github.com/katello/katello/commit/1ef1883))
+
+### API doc
+ * remove required flag from DELETE /subscriptions apipie ([#9984](http://projects.theforeman.org/issues/9984), [2cf41d8](http://github.com/katello/katello/commit/2cf41d8))
+
+### Docker
+ * Docker Content now hidden for composite cv ([#9610](http://projects.theforeman.org/issues/9610), [68312e2](http://github.com/katello/katello/commit/68312e2))
+ * Fix updating of docker upstream name ([#9423](http://projects.theforeman.org/issues/9423), [ab345f5](http://github.com/katello/katello/commit/ab345f5))
+ * Docker Content now hidden for composite cv ([#9610](http://projects.theforeman.org/issues/9610), [348c873](http://github.com/katello/katello/commit/348c873))
+ * Remove unused erroneous file ([#9539](http://projects.theforeman.org/issues/9539), [1c977c9](http://github.com/katello/katello/commit/1c977c9))
+
+ * fixing db/seeds.rb for katello, BZ 868910 ([#3272](http://projects.theforeman.org/issues/3272), [346f187](http://github.com/katello/katello/commit/346f187))
 
 ## Contributors
 
```

Comment:
This list looks like all contributors for all time, whereas in the past we were only generating the contributors list for that release to include in the release notes.